Doc/library/textwrap.rst

```diff
@@ -17,31 +17,69 @@ If you're just wrapping or filling one or two text strings, the convenience
 functions should be good enough; otherwise, you should use an instance of
 :class:`TextWrapper` for efficiency.
 
-.. function:: wrap(text, width=70, **kwargs)
+.. function:: wrap(text, width=70, initial_indent="", subsequent_indent="", \
+                   expand_tabs=True, replace_whitespace=True, \
+                   fix_sentence_endings=False, \
+                   break_long_words=True, drop_whitespace=True, \
+                   break_on_hyphens=True, tabsize=8, \
+                   *, max_lines=None)
 
    Wraps the single paragraph in *text* (a string) so every line is at most
    *width* characters long.  Returns a list of output lines, without final
-   newlines.
+   newlines.  *width* defaults to ``70``.
 
-   Optional keyword arguments correspond to the instance attributes of
-   :class:`TextWrapper`, documented below.  *width* defaults to ``70``.
+   See :class:`TextWrapper` for documentation on all additional optional
+   arguments:
 
-   See the :meth:`TextWrapper.wrap` method for additional details on how
-   :func:`wrap` behaves.
+   * :attr:`.initial_indent`
```

Comment:
I don't think these 3 listings are needed because the names are already in the sig and there's already a note referring to TextWrapper.

Reply:
The nice thing about that listing, though, is that it forms hyperlinks directly to each of these attributes, so it makes html browsing very smooth (see screenshot)

<img width="846" alt="Screen Shot 2021-07-02 at 3 38 15 PM" src="https://user-images.githubusercontent.com/58614260/124321338-86786780-db4b-11eb-938b-0e044eccea47.png">

Reply:
See 1306f2dd42328a9a903644363845621b5d961c1c; we've followed a precedent seen in the documentation for logging where the kwargs are listed out in prose-style.